gcloud/_apitools/http_wrapper.py

```diff
@@ -372,6 +321,53 @@ def _MakeRequestNoRetry(http, http_request, redirections=5,
     return response
 
 
+def MakeRequest(http, http_request, retries=7, max_retry_wait=60,
```

Comment:
Ah nice. I figured it was due to the dependence but didn't realize the change to the method was material.